tests/testfiles/CommonMark_0.30/backslash_escapes/010.test

```diff
@@ -8,9 +8,5 @@
 <a href="/bar\/)">
 >>>
 documentBegin
-BEGIN link
-- label: 
-- URI: /bar(backslash)/)
-- title: 
-END link
+inlineHtmlTag: <a href="/bar(backslash)/)">
```

Comment:
We currently don't translate HTML elements to their corresponding markdown equivalents. We just sanitize them and pass them to the renderers as-is.